APMrover2/AP_MotorsUGV.cpp

```diff
@@ -126,12 +126,12 @@ void AP_MotorsUGV::output(bool armed, float dt)
 void AP_MotorsUGV::output_regular(bool armed, float steering, float throttle)
 {
     // always allow steering to move
-    SRV_Channels::set_output_scaled(SRV_Channel::k_steering, steering);
+    output_throttle(SRV_Channel::k_steering, steering);
```

Comment:
we should rename this function. It is need otherwise the steering cannot use brushed and relay